code/modules/core_implant/cruciform/rituals/path.dm

```diff
@@ -0,0 +1,300 @@
+	stats_to_boost = list(STAT_VIG = 10)
+
+
+/datum/ritual/cruciform/monomial
+	name = "cruciform"
+	phrase = null
+	implant_type = /obj/item/weapon/implant/core_implant/cruciform/monomial
+	fail_message = "The Cruciform feels cold against your chest."
+	category = "Monomial"
+
+
+/datum/ritual/targeted/cruciform/monomial
+	name = "cruciform targeted"
+	phrase = null
+	implant_type = /obj/item/weapon/implant/core_implant/cruciform/monomial
+	category = "Monomial"
+
+/datum/ritual/cruciform/monomial/ironskin
+	name = "Resolution"
+	phrase = "Ut det Dominus virtutem populo suo? Sit Dominus benedicet populo suo in pace."
+	desc = "A short litany that removes all pain, it is much stronger then the relief litany, but requires more power and has a short cooldown between uses."
+	power = 50
+	cooldown_time = 5 MINUTES
+
+/datum/ritual/cruciform/monomial/ironskin/perform(mob/living/carbon/human/H, obj/item/weapon/implant/core_implant/C)
+	H.add_chemical_effect(CE_PAINKILLER, 10000, TRUE)
+	H.apply_effect(-200, AGONY, 0)
+	return TRUE
+
+/datum/ritual/cruciform/monomial/perfect_self
+	name = "Perfect Self"
+	phrase = "Quaerite primum regnum Dei et iustitiam eius, et haec omnia adicientur vobis."
+	desc = "A solemn prayer, often spoken in a whisper that allows a disciple of the monomial to focus inward and reach their full potential in both body and mind. This litany lasts ten minutes and takes over half an hour before it can be used again."
+	cooldown = TRUE
+	cooldown_time = 30 MINUTES
+	cooldown_category = "pself"
+	effect_time = 10 MINUTES
+	power = 90
+
+/datum/ritual/cruciform/monomial/perfect_self/perform(mob/living/carbon/human/user, obj/item/weapon/implant/core_implant/C)
+	user.stats.changeStat(STAT_TGH, 10)
+	user.stats.changeStat(STAT_ROB, 10)
+	user.stats.changeStat(STAT_VIG, 10)
+	user.stats.changeStat(STAT_COG, 10)
+	user.stats.changeStat(STAT_BIO, 10)
+	user.stats.changeStat(STAT_MEC, 10)
+	to_chat(user, SPAN_NOTICE("You feel at peace with yourself, your body and mind going beyond its limits."))
+	set_personal_cooldown(user)
+	addtimer(CALLBACK(src, .proc/discard_effect, user), src.cooldown_time)
+	return TRUE
+
+/datum/ritual/cruciform/monomial/perfect_self/proc/discard_effect(mob/living/carbon/human/user, amount)
+	user.stats.changeStat(STAT_TGH, -10)
+	user.stats.changeStat(STAT_ROB, -10)
+	user.stats.changeStat(STAT_VIG, -10)
+	user.stats.changeStat(STAT_COG, -10)
+	user.stats.changeStat(STAT_BIO, -10)
+	user.stats.changeStat(STAT_MEC, -10)
+
+
+/datum/ritual/cruciform/divisor
+	name = "cruciform"
+	phrase = null
+	implant_type = /obj/item/weapon/implant/core_implant/cruciform/divisor
+	category = "Divisor"
+
+
+/datum/ritual/targeted/cruciform/divisor
+	name = "cruciform targeted"
+	phrase = null
+	implant_type = /obj/item/weapon/implant/core_implant/cruciform/divisor
+	category = "Divisor"
+
+/datum/ritual/targeted/cruciform/divisor/spawn_con
+	name = "Canticle of Defense"
```

Comment:
How does this one even work?
Like, im not going "grrr church op"
or a code question of that kind
Rather in character, does the nanites in the cruciform nit together all the things every 4 hours?